src/test/java/org/folio/rest/api/LoansApiTest.java

```diff
@@ -68,6 +69,43 @@ public void checkIdsAfterEach() {
     StorageTestSuite.checkForMismatchedIDs("loan");
   }
 
+  @Test
+  public void canAnonymizeLoans() throws InterruptedException, ExecutionException, TimeoutException, MalformedURLException {
+
+    UUID itemId = UUID.randomUUID();
+    UUID userId = UUID.randomUUID();
+
+    JsonObject loan1 = loansClient.create(new LoanRequestBuilder()
+      .withId(UUID.randomUUID())
+      .withItemId(itemId)
```

Comment:
Is it possible to replace _itemId_ and _userId_ with UUID.randomUUID()?
Or do you plan to use these variables somewhere in the code?

Reply:
Done

Reply:
This has been reworked.